app/integration_test/support/mail.dart

```diff
@@ -111,7 +111,7 @@ extension ActerMail on ConvenientTest {
   }) async {
     var link = await getLinkInLatestEmail(emailAddr, contains: contains);
     if (replaceWithHomeserver) {
-      final homeserver = Uri.parse(defaultServerUrl);
+      final homeserver = Uri.parse(Env.defaultHomeserverUrl);
```

Comment:
All occurrences of `String.fromEnvironment()` or `bool.fromEnvironment()` should be replaced with `enven`?

Reply:
Yes, they are not built with the environment anymore but from `Env` now.